root/report/recordings_with_the_same_name_by_different_artists_with_the_same_name.tt

```diff
@@ -0,0 +1,17 @@
+[%- WRAPPER 'layout.tt' title=l('Recordings with the same name by different artists with the same name') full_width=1 -%]
+
+<h1>[% l('Recordigns with the sae name by different artists with the same name') %]</h1>
+
+<ul>
+    <li>[% l('This report shows all recordings with the same name that have different artist MBIDs with the same name.') %]</li>
```

Comment:
I would say: "This report lists recordings with the same name that are credited to different artists with the same name."